_blog.yml

```
@@ -4721,4 +4721,13 @@
     - research
     - leaderboard
     - LLM
-    
\ No newline at end of file
+    
+- local: improve_parquet_dedupe
+  title: "Improving Parquet Dedupe on Hugging Face Hub"
+  author: yuchenglow
+  thumbnail: /blog/assets/improve_parquet_dedupe/thumbnail.png
+  date: Oct 5, 2024
```

```suggestion
  date: October 5, 2024
```

Just for consistency, not sure it matters for parsing and sorting the entries.